featurebyte/session/base.py

```diff
@@ -496,7 +496,7 @@ async def execute_query(
         return dataframe_from_arrow_stream(data)
 
     async def execute_query_interactive(
-        self, query: str, timeout: float = INTERACTIVE_SESSION_TIMEOUT_SECONDS
+        self, query: str, timeout: float = DEFAULT_EXECUTE_QUERY_TIMEOUT_SECONDS
     ) -> pd.DataFrame | None:
         """
         Execute SQL query that is expected to run for a short time
```

Comment:
Is `DEFAULT_EXECUTE_QUERY_TIMEOUT_SECONDS` considered short enough? What is the consequence if the query ended up running for a long time when calling this method?

Reply:
Looks like `DEFAULT_EXECUTE_QUERY_TIMEOUT_SECONDS` is for `def execute_query()`. Will revert this. Setting the timeout too long for interactive query can cause threads to accumulate when traffic to the api service is heavy